### PR TITLE
gh17742 new GH action and script to create a...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

gh17742 new GH action and script to create a branch once every 7.yy.0 release (to be called by che-release script)

Change-Id: Ie192d6969c522317bdc030c12260e0d754e3033b
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17742

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/make-branch.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    name: Create branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create branch
+        run: |
+          git config --global user.name "Mykhailo Kuznietsov"
+          git config --global user.email "mkuznets@redhat.com"
+          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
```
Comment (mkuznyetsov) [inline .github/workflows/make-branch.yaml:20]: this is unnecessary, as the GITHUB_TOKEN variable already exists and points to that same value (GH actions bot token)

File: .github/workflows/make-branch.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          .github/bin/make-branch.sh --branch ${{ github.event.inputs.branch }}
```
Comment (mkuznyetsov) [inline .github/workflows/make-branch.yaml:21]: is this script supposed to be included in this PR too?

**Review by mkuznyetsov — CHANGES_REQUESTED** [review]

File: .github/workflows/make-branch.yaml (hunk 1 above)
Comment (nickboldt) [reply .github/workflows/make-branch.yaml:20]: I copied this from https://github.com/eclipse/che-machine-exec/blob/master/.github/workflows/release.yaml#L35-L38

File: .github/workflows/make-branch.yaml (hunk 2 above)
Comment (nickboldt) [reply .github/workflows/make-branch.yaml:21]: yes, good catch. :D

File: .github/workflows/make-branch.yaml (hunk 1 above)
Comment (mkuznyetsov) [reply .github/workflows/make-branch.yaml:20]: ok, then it can stay ... or not 😄

**Review by mkuznyetsov — APPROVED** [review]

**Pull request merged**